towerlib/entities/host.py

```diff
@@ -283,24 +283,24 @@ def recent_jobs(self):
 
     @property
     def ansible_facts(self):
-        """Returns ansible facts gathered about the host in json.
+        """Ansible facts gathered about the host in json.
 
         Args: None
 
         Returns:
             json: Json representation of ansible facts
 
         Raises: None
+
         """
         url = '{api}/hosts/{id}/ansible_facts'.format(
             api=self._tower.api, id=self.id)
         response = self._tower.session.get(url)
-        if response.ok:
-            return response.json
-        else:
+        if not response.ok:
             self._logger.error('Error finding ansible facts for {name}.'.format(
                 name=self._data.get('name')))
-            return '{}'
+        return response.json if response.ok else '{}'
```

Comment:
response.json is a method. you probably want to return response.json()

Reply:
Yes i just changed the logic and over-looked that it was already wrong there. Will fix.